x-pack/platform/plugins/shared/osquery/server/plugin.ts

```diff
@@ -151,10 +154,12 @@ export class OsqueryPlugin implements Plugin<OsqueryPluginSetup, OsqueryPluginSt
         if (registerIngestCallback) {
           registerIngestCallback(
             'packagePolicyCreate',
-            async (newPackagePolicy: NewPackagePolicy): Promise<UpdatePackagePolicy> => {
+            async (
+              newPackagePolicy: NewPackagePolicy,
+              soClient: SavedObjectsClientContract
+            ): Promise<UpdatePackagePolicy> => {
               if (newPackagePolicy.package?.name === OSQUERY_INTEGRATION_NAME) {
```

Comment:
@nchaulet 
It works and I believe that's what you had on your mind, correct? :)